6/melinda-seckington-captions.srt

```diff
@@ -513,7 +514,7 @@ DN: Oh that's...
 
 119
 00:08:21,072 --> 00:08:25,056
-MS: Yes, sublime, like soup, try to open
+MS: Yes, sublime, likes to, try to open...
```

Comment:
Ohhps - I thought soup might be wrong but then I do hear some strange words in the programming language!